core-bundle/src/Search/Indexer/DefaultIndexer.php

```diff
@@ -122,9 +122,6 @@ public function clear(): void
         $this->connection->executeStatement('TRUNCATE TABLE tl_search_term');
     }
 
-    /**
-     * @throws IndexerException
-     */
```

Comment:
Maybe use `@return never` here?
_(Just a nitpick, can be ignored ☺️)_

Reply:
Added in 4b170582f71bd8906902e2b5fd5ef2205966ae50.